tests/wpt/metadata/custom-elements/custom-element-registry/define.html.ini

```diff
@@ -1,5 +1,53 @@
 [define.html]
   type: testharness
-  [Custom Elements: Element definition]
+  [If constructor is arrow function, should throw a TypeError]
```

Comment:
I am not sure how I should check for this, is there anything other than `IsConstructor` that I need to do?